homeassistant/components/epsonworkforceprinter/sensor.py

```diff
@@ -0,0 +1,85 @@
+"""Support for Epson Workforce Printer."""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_HOST, CONF_MONITORED_CONDITIONS
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['epsonprinter==0.0.8']
+
+_LOGGER = logging.getLogger(__name__)
+DEFAULT_IP = '127.0.0.1'
+MONITORED_CONDITIONS = {
+    'black': ['Inklevel Black', '%', 'mdi:water'],
+    'magenta': ['Inklevel Magenta', '%', 'mdi:water'],
+    'cyan': ['Inklevel Cyan', '%', 'mdi:water'],
+    'yellow': ['Inklevel Yellow', '%', 'mdi:water'],
+    'clean': ['Inklevel Cleaning', '%', 'mdi:water'],
+}
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST, default=DEFAULT_IP): cv.string,
```

Comment:
If it's required then there should be no default.

Reply:
Okay, removed it